dev/breeze/README.md

```diff
@@ -62,6 +63,67 @@ available when the package was installed. Since this file becomes part of the in
 to detect automatically if any of the files have changed. If they did, the user will be warned to upgrade
 their installations.
 
+Setting up development env for Breeze
+-------------------------------------
+
+Breeze is actively developed by Airflow maintainers and contributors, Airflow is an active project
+and we are in the process of developing Airflow 3, so breeze requires a lot of adjustments to keep up
+the dev environment in sync with Airflow 3 development - this is also why it is part of the same
+repository as Airflow - because it needs to be closely synchronized with Airflow development.
+
+As of November 2024 Airflow switches to using `uv` as the main development environment for Airflow
+and for Breeze. So the instructions below are for setting up the development environment for Breeze
+using `uv`. However we are using only standard python packaging tools, so you can still use `pip` or
+`pipenv` or other build frontends to install Breeze, but we recommend using `uv` as it is the most
+convenient way to install, manage python packages and virtual environments.
+
+Unlike in Airflow, where we manage our own constraints, we use `uv` to manage requirements for Breeze
+and we use `uv` to lock the dependencies. This way we can ensure that the dependencies are always
+up-to-date and that the development environment is always consistent for different people. This is
+why Breeze's `uv.lock` is committed to the repository and is used to install the dependencies by
+default by Breeze.
```

Comment:
Do we have to switch the directory to dev?

Reply:
Shouldnt be needed but it would probably be more "complete" that way

Reply:
This is all about being in "dev/breeze" folder - I will make it celar in the docs,

Reply:
Oh.. I found out that we have this really nice markdown extension available in Github - to mark notes: 

<img width="1188" alt="Screenshot 2024-11-04 at 13 16 16" src="https://github.com/user-attachments/assets/12fc0286-6b37-40b7-9fbe-82fbc84dfd02">

We should definitely apply it in more of our docs :)

Reply:
super cool :)

Reply:
Oh this is nice